hw2/03-count.js

```diff
@@ -1,3 +1,39 @@
 // Add your code here
+const searchWord = document.getElementById("search-word");
+const content = document.getElementById("content");
+const sentence = document.getElementById("content").innerHTML;
+const regex = new RegExp(searchWord, "gi");
 
-input.addEventListener('keydown', handleKeyDown);
+const findAndHighlight = function findAndHighlight() {
+  if (searchWord.value !== "") {
+    let words = content.innerHTML.split(" ");
+    for (let i = 0; i < words.length; i++) {
+      if (words[i] === searchWord.value) {
```

Comment:
This logic misses some cases where words have punctuation since you're splitting the original text on whitespace. For example, the program will fail to highlight if you input `Light` or `Heaven`, since within your `words[]` array the target words are stored as `Light,` and `Heaven,`